pkg/apis/crd/v1beta1/types.go

```diff
@@ -1004,9 +1004,17 @@ type EgressSpec struct {
 	// Cannot be set with ExternalIPPool.
 	ExternalIPPools []string `json:"externalIPPools,omitempty"`
 	// Bandwidth specifies the rate limit of north-south egress traffic of this Egress.
-	Bandwidth *Bandwidth `json:"bandwidth,omitempty"`
+	Bandwidth     *Bandwidth        `json:"bandwidth,omitempty"`
+	FailurePolicy FailurePolicyType `json:"failurePolicy"`
```

Comment:
Shouldn't it be a pointer given existing egress won't have it set?

And what's the default value? Please add proper comment for this new field for better understanding.

Reply:
Default would be NodeSNAT

Reply:
added coment